docs/install/assets/eventing.md

```diff
@@ -0,0 +1,11 @@
+# Knative Serving installation files (0.21 releases)
+
+| File name | Description | Dependencies|
+| --- | --- | --- |
+| eventing-core.yaml | Knative Serving core components. |  none |
```

Comment:
replace `none` with `eventing-crds.yaml`

Reply:
I might be missing something but `eventing-core.yaml` includes the CRDs such as `apiserversources.sources.knative.dev`, `brokers.eventing.knative.dev`, etc 
So, is `eventing-crds.yaml` a dependency ?

Reply:
Yes we like to say is a dependency because most of the time installing core.yaml fails because of eventual consistency and race conditions. This is why in the instructions we tell to install cards yaml first once that's settle then install core yaml even if it contains the same cards again

Reply:
Thanks